doc/src/guides/solving/solve-system-of-equations-algebraically.md

```diff
@@ -1,29 +1,15 @@
 # Solve a System of Equations Algebraically
 
-Use SymPy to algebraically solve a system of equations, whether linear or
-nonlinear. For example, solving $x^2 + y = 2z, y = -4z$ for x and y (assuming z
-is a constant or parameter) yields $\{(x = -\sqrt{6z}, y = -4z),$ ${(x =
-\sqrt{6z}, y = -4z)\}}$.
+Use SymPy to solve a system of equations algebraically. For example, solving
+$x^2 + y = 2z, y = -4z$ for x and y (assuming z is a constant or parameter)
+yields $\{(x = -\sqrt{6z}, y = -4z),$ ${(x = \sqrt{6z}, y = -4z)\}}$.
```

Comment:
Why is this file modified? Did you resolve a merge conflict incorrectly?

Reply:
It's possible, but I'm not sure. My repo before the forced merge was about 192 commits behind the main repo.

Reply:
Just copied the contents of the file in master and pasted it into the same file in my branch. Does this fix this issue?